time tiempoEjecucion
int32 anguloPitch
int32 anguloRoll
int32 anguloYaw
int32 G_Pitch
int32 G_Roll
int32 G_Yaw
int32 G_Pitch_filtrada
int32 G_Roll_filtrada
int32 G_Yaw_filtrada
int32 Ax
int32 Ay
int32 Az
int32 Ax_filtrada
int32 Ay_filtrada
int32 Az_filtrada
int32 posZ
int32 posZ_filtrada
int32 velZ
int32 motorDelantero
int32 motorTrasero
int32 motorDerecho
int32 motorIzquierdo
int32 encendido
int64 mensajesRecibidos
